### ISSUE TITLE
get the attributes

### ISSUE THREAD

**Issue by auqf** [uzresk/jmx-cmdclient#3]
how can i know the attributes of the bean

**Comment by larsga** [comment]
Yes, my exact question as well. I wanted to use this client to learn what JMX values a certain service exposes. I can find the beans, but not their attributes, so ultimately this doesn't help me. With that tiny addition this would have been a very helpful tool.
